engine/requirements.cpp

```diff
@@ -321,6 +341,13 @@ engine::check_reqs(const model::metadata& md, const config::tree& cfg,
     if (!reason.empty())
         return reason;
 
+    // Iterate over extra checkers registered.
+    for (auto checker : engine::reqs_checkers()) {
```

Comment:
Best to pass by reference instead of by copy.
```suggestion
    for (auto& checker : engine::reqs_checkers()) {
```

Reply:
Nice spot. Fixed.